pkg/apis/v1alpha5/maintenance_window.go

```diff
@@ -0,0 +1,67 @@
+/*
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package v1alpha5
+
+import (
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+// MaintenanceWindowSpec is the top level field for MaintenanceWindows.
+type MaintenanceWindowSpec struct {
```

Comment:
Discussed offline. I think this name implies the inverse semantic. We discussed `NodeDisruptionGate`

Reply:
I'll move this and the RFC to NodeDisruptionGate, and see if there any complaints. Maybe MachineDisruptionGate here? @jonathan-innis